nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -59,7 +59,7 @@ BtNavigator::BtNavigator()
   };
 
   // Declare this node's parameters
-  declare_parameter("bt_xml_filename");
+  declare_parameter("default_bt_xml_filename", std::string(""));
```

Comment:
Remove the default, we'd rather it throw an exception than try to open a bogus filepath, don't you think? I think the majority of applications may not give a filepath so the default would be a required variable.

Reply:
ok